### PR TITLE
Removed tick arrow within spinner slider + rounded handle

### LINKED ISSUE #5358: Arrow is broken
Arrow is broken in tooltip 

image

### PR DESCRIPTION
**Pull request by xavijam**
screen shot 2015-09-08 at 11 10 55

Fixes #5358 
REVIEWER: @javierarce

### CODE REVIEW

File: app/assets/stylesheets/old_elements/custom_objects.css.scss [hunk 1]
```diff
@@ -33,6 +33,12 @@
     padding-bottom:10px;
     height:81px;
 
+    // Stop adding tick arrow
+    // Overriding default dropdown before value
+    &:after,
+    &.border:after,
+    &:before { display: none!important }
```
Comment (xavijam) [inline app/assets/stylesheets/old_elements/custom_objects.css.scss]: Unfortunately, as we have the CSS in the editor part, we have to appeal to these things :(.

File: app/assets/stylesheets/old_elements/custom_objects.css.scss [hunk 2]
```diff
@@ -58,7 +64,7 @@
         background:#397DBA;
         border:1px solid #255078;
         z-index: 2;
-        @include border-radius(4px);
+        border-radius: 10px;
```
Comment (xavijam) [inline app/assets/stylesheets/old_elements/custom_objects.css.scss]: A proper rounded handle

**Comment by xavijam** [comment]
friendly ping @javierarce :D

**Comment by javierarce** [comment]
:+1: :+1: :+1:

**Pull request merged**